lib/tls.js

```diff
@@ -231,7 +231,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 // Example:
 // C=US\nST=CA\nL=SF\nO=Joyent\nOU=Node.js\nCN=ca1\nemailAddress=ry@clouds.org
 exports.parseCertString = function parseCertString(s) {
-  var out = {};
+  var out = Object.create(null);
```

Comment:
@mcollina `parseCertString()` isn't documented.  You might be thinking of #14193.